docs/getting-started/configuration.rst

```diff
@@ -21,124 +21,415 @@ Below are all the available options, simple examples can be found in the docker-
 
 .. _docker-sync-boilerplate: https://github.com/EugenMayer/docker-sync-boilerplate
 
-References
-----------
 
-.. caution::
-
-    This is a configuration reference. **Do not use all options at once, they do not make sense!** Or copy them as your starting point, rather use the docker-sync-boilerplate and then cherry pick the options you need.
+Simple configuration file
+-------------------------
 
 .. code-block:: yaml
 
     version: "2"
     options:
-      # default: docker-compose.yml if you like, you can set a custom location (path) of your compose file like ~/app/compose.yml
-      # HINT: you can also use this as an array to define several compose files to include. Order is important!
-      compose-file-path: 'docker-compose.yml'
+      verbose: true
+    syncs:
+      appcode-native-osx-sync:
+        src: './app'
+        sync_excludes: ['ignored_folder', '.ignored_dot_folder']
 
-      # optional, default: docker-compose-dev.yml if you like, you can set a custom location (path) of your compose file. Do not set it, if you do not want to use it at all
 
-      # if its there, it gets used, if you name it explicitly, it HAS to exist
-      # HINT: you can also use this as an array to define several compose files to include. Order is important!
-      compose-dev-file-path: 'docker-compose-dev.yml'
+General options
+---------------
+Configuration options below the top-level ``options`` key.
```

Comment:
we should add a visual example here, what do you think?

Reply:
Done.